x-pack/plugins/endpoint/public/applications/endpoint/store/hosts/selectors.ts

```diff
@@ -81,7 +83,7 @@ export const policyResponseFailedOrWarningActionCount: (
  */
 export const policyResponseActions: (
   state: Immutable<HostState>
-) => undefined | Partial<HostPolicyResponseActions> = createSelector(
+) => undefined | ImmutableArray<HostPolicyResponseAppliedAction> = createSelector(
```

Comment:
Just use `Immutable<>` instead of `ImmutableArray<>`. That type will take care of using the appropriate Immutable type in its implementation. I saw Rob provide this feedback recently when I saw him providing the same feedback - the reason being: if Typescript introduces a builtin `Immutable` Generic, then we will be in a better place to remove our implementation.